### PR TITLE
Execute the Run function of kubelet, no log output after failure

### LINKED ISSUE #110690: Execute the Run function of kubelet, no log output after failure
### What happened?

When running kubelet, if the Run function fails, the returned error has no output, and the cause of the error cannot be located.
https://github.com/kubernetes/kubernetes/blob/a9f83d839ec311ceaf4ef8ed35dbdfcff8ae0b3a/cmd/kubelet/app/server.go#L268

such as 
https://github.com/kubernetes/kubernetes/blob/a9f83d839ec311ceaf4ef8ed35dbdfcff8ae0b3a/pkg/kubelet/cm/cgroup_manager_linux.go#L278
After the error is returned, there is no position output, so the user cannot know the cause of the error.

But previous versions had log output
https://github.com/kubernetes/kubernetes/blob/938a3203c609d42b61512a55f42344ce1438f5a9/pkg/kubelet/cm/cgroup_manager_linux.go#L300

https://github.com/kubernetes/kubernetes/blob/938a3203c609d42b61512a55f42344ce1438f5a9/cmd/kubelet/app/server.go#L301

### What did you expect to happen?

Run the Run function, when there is an error exit, print the error log

### How can we reproduce it (as minimally and precisely as possible)?

Misconfigured cgroupRoot

### Anything else we need to know?

_No response_

### Kubernetes version

<details>

```console
$ kubectl version
# paste output here
root@e2e-control-plane:/#  kubectl version
WARNING: This version information is deprecated and will be replaced with the output from kubectl version --short.  Use --output=yaml|json to get the full version.
Client Version: version.Info{Major:"1", Minor:"24", GitVersion:"v1.24.1", GitCommit:"3ddd0f45aa91e2f30c70734b175631bec5b5825a", GitTreeState:"clean", BuildDate:"2022-05-24T12:26:19Z", GoVersion:"go1.18.2", Compiler:"gc", Platform:"linux/amd64"}
Kustomize Version: v4.5.4
Server Version: version.Info{Major:"1", Minor:"24", GitVersion:"v1.24.1", GitCommit:"3ddd0f45aa91e2f30c70734b175631bec5b5825a", GitTreeState:"clean", BuildDate:"2022-05-24T12:18:48Z", GoVersion:"go1.18.2", Compiler:"gc", Platform:"linux/amd64"}


```

</details>

### Cloud provider

<details>

</details>

### OS version

<details>

```console
# On Linux:
$ cat /etc/os-release
# paste output here
$ uname -a
# paste output here

# On Windows:
C:\> wmic os get Caption, Version, BuildNumber, OSArchitecture
# paste output here
```

</details>

### Install tools

<details>

</details>

### Container runtime (CRI) and version (if applicable)

<details>

</details>

### Related plugins (CNI, CSI, ...) and versions (if applicable)

<details>

</details>

### PR DESCRIPTION
**Pull request by yangjunmyfm192085**
#### What type of PR is this?
/kind bug

#### What this PR does / why we need it:
run the Run function, when there is an error exit, print the error log
As discussed in https://github.com/kubernetes/kubernetes/pull/110691#discussion_r911652960. We need to print the log before `return 1`
#### Which issue(s) this PR fixes:

Fixes #110690

#### Does this PR introduce a user-facing change?

```release-note
Run kubelet, when there is an error exit, print the error log
```

#### Additional documentation e.g., KEPs (Kubernetes Enhancement Proposals), usage docs, etc.:

```docs

```

### CODE REVIEW

**Review by yangjunmyfm192085 — COMMENTED** [review]
/retest

File: cmd/kubelet/kubelet.go [hunk 1]
```diff
@@ -53,6 +54,7 @@ func run(command *cobra.Command) int {
 
 	command.SetGlobalNormalizationFunc(cliflag.WordSepNormalizeFunc)
 	if err := command.Execute(); err != nil {
+		fmt.Fprintf(os.Stderr, "Error: %v\n", err)
```
Comment (pohly) [inline cmd/kubelet/kubelet.go:57]: /lgtm

It's worth pointing out that this doesn't attempt to determine *how* the error should be printed (plain text or log entry), but that is good enough here. `cli.Run` on master handles that.

**Review by yangjunmyfm192085 — COMMENTED** [review]
/triage accepted
/assign @mrunalp

**Review by mrunalp — APPROVED** [review]

**Review by yangjunmyfm192085 — COMMENTED** [review]
/cc kubernetes/release-managers

**Review by yangjunmyfm192085 — COMMENTED** [review]
/assign @justaugustus

**Review by justaugustus — APPROVED** [review]

**Pull request merged**